packages/jest-each/src/bind.js

```diff
@@ -85,16 +86,15 @@ const arrayFormat = (title, ...args) => {
       if (prettyIndexes.indexOf(index) !== -1) {
         return {
           args: acc.args,
-          title: acc.title.replace(
-            PRETTY_PLACEHOLDER,
-            pretty(arg, {maxDepth: 1, min: true}),
-          ),
+          title: acc.title
+            .replace(PRETTY_PLACEHOLDER, pretty(arg, {maxDepth: 1, min: true}))
+            .replace(INDEX_PLACEHOLDER, `${rowIndex}`),
```

Comment:
I think this and line 97 can move outside of the `reduce` and just do a `replace` on the `prettyTitle` returned before calling `util.format` on line 103.

Maybe something like:

```js
return util.format(
    prettyTitle.replace(INDEX_PLACEHOLDER, rowIndex.toString()),
    ...remainingArgs.slice(0, placeholders.length - prettyIndexes.length),
  );
```

Reply:
@mfix22 anything preventing us from applying this solution? Seems more generic approach

Reply:
Oooh i missed this. I'll take a look.